deltachat-rpc-client/tests/test_something.py

```diff
@@ -103,12 +103,12 @@ def test_account(acfactory) -> None:
     assert alice.get_chatlist(snapshot=True)
     assert alice.get_qr_code()
     assert alice.get_fresh_messages()
-    assert alice.get_next_messages()
```

Comment:
This check isn't reliable anymore. Before some sync messages were returned, but now they are sent a bit later (from the SMTP loop). Looks like this check is only to test the API, so i moved it below so that bob checks that `get_next_messages()` return the same as `wait_next_messages()`.